libid/complex.cpp

```diff
@@ -0,0 +1,536 @@
+// SPDX-License-Identifier: GPL-3.0-only
+//
+// Complex.cpp: interface for the Complex class.
+//
+//////////////////////////////////////////////////////////////////////
+#include "Complex.h"
+
+//Complex::Complex(void)
+//    {
+//    }
+
+Complex::~Complex(void)
+    {
+    }
+
+Complex Complex::operator=(const double & value)	// Assignment to double Operator
+    {
```

Comment:
Outdent braces here and throughout.  It is probably going to be easiest just to select the entire file in Visual Studio and select "Format Selection" from the context menu.

clang-format will take care of everything.